lighthouse-core/test/lib/icons-test.js

```diff
@@ -36,6 +36,17 @@ describe('Icons helper', () => {
       assert.equal(icons.doExist(manifest.value), false);
     });
 
+    it('fails when a manifest contains no fetchable icons', () => {
```

Comment:
ooo ooo can we get an empty string test in here before we forget about it again :D